danieldupriest/reimbursinator
# added conditional to check user in finalize_report()

Updated code to ensure a user can't manipulate other reports.

TESTING:
1) update `.env` details
2) run `pipenv` commands, and `python manage.py test`
3) ensure all tests pass

Closes #127
